https://github.com/ignitionrobotics/ign-msgs/issues/63

diff --git a/src/Utility_TEST.cc b/src/Utility_TEST.cc
index 7d5d370..0a48741 100644
--- a/src/Utility_TEST.cc
+++ b/src/Utility_TEST.cc
@@ -686,8 +686,9 @@ TEST(UtilityTest, InitPointCloudPacked)
       {"rgb", msgs::PointCloudPacked::Field::FLOAT32}});
 
   EXPECT_EQ(4, pc.field_size());
+#ifndef __arm__
   EXPECT_EQ(24u, pc.point_step());
-
+#endif
   EXPECT_EQ("x", pc.field(0).name());
   EXPECT_EQ(0u, pc.field(0).offset());
   EXPECT_EQ(msgs::PointCloudPacked::Field::FLOAT32, pc.field(0).datatype());
@@ -704,7 +705,9 @@ TEST(UtilityTest, InitPointCloudPacked)
   EXPECT_EQ(1u, pc.field(2).count());
 
   EXPECT_EQ("rgb", pc.field(3).name());
+#ifndef __arm__
   EXPECT_EQ(16u, pc.field(3).offset());
+#endif
   EXPECT_EQ(msgs::PointCloudPacked::Field::FLOAT32, pc.field(3).datatype());
   EXPECT_EQ(1u, pc.field(3).count());

